components/identity-mgt/org.wso2.carbon.identity.mgt.endpoint.util/src/main/java/org/wso2/carbon/identity/mgt/endpoint/util/IdentityManagementEndpointUtil.java

```diff
@@ -641,4 +664,57 @@ public static void setOptions(ServiceClient client, String accessUsername, Strin
 
         IdentityManagementServiceUtil.setAutheticationOptions(client, accessUsername, accessPassword);
     }
+
+    /**
+     * Get base path URL for API clients.
+     *
+     * @param tenantDomain tenant Domain.
+     * @param context      URL context.
+     * @return base path.
+     * @throws ApiException ApiException.
+     */
+    public static String getBasePath(String tenantDomain, String context) throws ApiException {
+
+        String basePath = null;
+        try {
+            if (IdentityTenantUtil.isTenantQualifiedUrlsEnabled()) {
+                basePath = ServiceURLBuilder.create().addPath(context).build().getAbsoluteInternalURL();
+            } else {
+                if (!MultitenantConstants.SUPER_TENANT_DOMAIN_NAME.equalsIgnoreCase(tenantDomain)) {
+                    basePath = IdentityManagementEndpointUtil.buildServerUrl(tenantDomain, context);
```

Comment:
We are calling an internal method any way right. No need of using 'IdentityManagementEndpointUtil'

Reply:
Yes. Updated the PR accordingly.